onnx/compose.py

```diff
@@ -206,12 +205,22 @@ def merge_graphs(
     g.node.extend(g2.node)
     g2_nodes_end = len(g.node)
 
+    # Searching inputs of the subgraph recursively.
```

Comment:
Pull this out to the module name space to avoid nested functions (hard to test. Use nested functions only when it is closing over local variables) and mark as private by prepending `_` in the function name?

Reply:
I think this uses `reversed_io_map` which is a local variable. This is also not meant to be tested externally ... does that make a difference?

Reply:
I missed reversed_io_map. If that's the case we should be good.